third_party/move/move-compiler-v2/tests/checking/typing/vector_with_non_base_type.exp

```diff
@@ -0,0 +1,49 @@
+
+Diagnostics:
+error: invalid type instantiation `&u64`: only structs, vectors, and primitive types allowed
+  ┌─ tests/checking/typing/vector_with_non_base_type.move:4:13
+  │
+4 │         let v = vector<&u64>[];
+  │             ^
+
+error: invalid type instantiation `&u64`: only structs, vectors, and primitive types allowed
```

Comment:
This error is similar to the one above, the only difference is the location info. @wrwg